### ISSUE TITLE
Update model to include dirty mat images

### ISSUE THREAD

**Issue by mkutu** [precision-sustainable-ag/Field-AnnotationPipeline#91]
- [x] 1. find images of dirty mats (the image below is from (TX_2023-10-06, ragweed parthenium)
- [ ] 2. create prediction mask using SAM/SAM-HQ
- [x] #96
~~- [ ] 3. clean up prediction masks in CVAT~~
- [x] 4. Update current model by training on these 100 images
    - load in current weight file
    - train
    - output second "updated" weight file that can then be used for inferencing
- [x] 5. run predictions across this and various other species to insure model is still accurate across other species

Notes:
When updating the model, we need to be sure that the segmentation masks on other classes don't lose accuracy. If updating the model simply by training on these 100 image does not work, update by pulling all the data and training from scratch. 

Include image to correct for this type of edge case:
![Image](https://github.com/user-attachments/assets/d7916460-db26-4b03-b8c9-dc70592c19ba)

**Comment by mkutu** [comment]
might replace with deeplabv3plus if we can't make any progress here

**Comment by navjot-nangia** [comment]
@mkutu 
Here are results from the newly trained UNET model with images + masks for dirty masks. 102 images + cleaned masks were added to the training data. 

I decided to retrain the model instead of fine-tuning. 


<img width="1470" alt="Image" src="https://github.com/user-attachments/assets/04225f09-2f78-4162-bc8b-270ddd797cfb" />
<img width="1470" alt="Image" src="https://github.com/user-attachments/assets/9add172d-895b-47e5-8ac3-c6a0a84c3ff9" />
<img width="1470" alt="Image" src="https://github.com/user-attachments/assets/f40ce01b-8fc1-41f3-b5d7-12998c5006e0" />
<img width="1470" alt="Image" src="https://github.com/user-attachments/assets/64d37a29-d025-4df9-a915-364b9bd1178b" />
<img width="1470" alt="Image" src="https://github.com/user-attachments/assets/299b2f91-e98f-4166-bc67-17cf32392d1f" />
<img width="1470" alt="Image" src="https://github.com/user-attachments/assets/d9a71901-b7c2-4146-9037-038c4ed401f9" />
<img width="1470" alt="Image" src="https://github.com/user-attachments/assets/2b57e13d-a1c3-40f2-b2a2-e8922cad4a70" />
